### ISSUE TITLE
Quick Paranoid Audit

### ISSUE THREAD

**Issue by jackturnbull** [pasupulaphani/libpostal-docker#4]
Hi and thanks for the work. We're looking at using this in a deployment of ours and gives us a good base to work from. Since there aren't too many people currently using it (safety in numbers), is it okay if I ask some questions or propose some changes of the sources used by the image? Happy to make any of the changes myself but I thought it'd be worth asking the questions first 😄 

I'll list some down here:

What is the purpose of using the CZ Ubuntu source? Can this image be used without an additional source? [LOC](https://github.com/pasupulaphani/libpostal-docker/blob/master/Dockerfile#L14):
```dockerfile
RUN echo "deb http://cz.archive.ubuntu.com/ubuntu trusty main" > /etc/apt/sources.list
```

Although I know there's an incredibly low chance of this file containing anything executable (and then that being actually ran), do you think it'd be a good idea to check this against a known good SHA-256? [LOC](https://github.com/pasupulaphani/libpostal-docker/blob/master/Dockerfile#L33):
```dockerfile
RUN wget https://libpostal.s3.amazonaws.com/mapzen_sample/parser_full.tar.gz
```

Can Node be installed using a more [standard means](https://github.com/nodejs/docker-node/blob/master/Dockerfile.template) than through s3pository, since not everyone will see the benefits as they might not be on Heroku? [LOC](https://github.com/pasupulaphani/libpostal-docker/blob/master/Dockerfile.zeromq#L33):
```dockerfile
RUN curl -s https://s3pository.heroku.com/node/v$NODE_ENGINE/node-v$NODE_ENGINE-linux-x64.tar.gz | tar --strip-components=1 -xz -C $NODE_PATH
```

Thanks again!

**Comment by pasupulaphani** [comment]
Hi @jackturnbull , Sorry for the delayed response. I'm just seeing this issue. I agree with all the above mentions and can be improved. I don't see any reason why not to get from secured sources if any available.

Please feel free to make a PR. Let me know if there is anything I can assist you with.

**Comment by jackturnbull** [comment]
@pasupulaphani sorry dude, not using the implementation any more so I'm not gonna find time to get round to it - it wasn't related to any of the above points though, we've solved our problem without libpostal. Happy to leave the issue open in case it's useful to anyone else.

**Comment by pasupulaphani** [comment]
@jackturnbull you found any alternative to libpostal? (asking just out of curiosity)

**Comment by jackturnbull** [comment]
We didn't in the end, we've just decided to offer 2 fields for the users; address and postcode. We don't do any filtering on the address fields themselves, but set coordinates using the postcode. Address is only for searching so we've dumped that into a Postgres TSVector.

Our spot tests of some of our real data didn't give us the accuracy we needed, even with the latest training data so we changed requirements and decided to cut the task out completely.
